for logistics reasons we want the debuginfo to actually be packaged in /usr/share/debug

--- rpm-4.12.0.1/scripts/find-debuginfo.sh~	2017-01-14 03:12:35.000000000 +0000
+++ rpm-4.12.0.1/scripts/find-debuginfo.sh	2019-05-04 14:20:57.748140126 +0000
@@ -371,3 +371,5 @@
   cat "$LISTFILE" >> "${LISTFILE}.new"
   mv "${LISTFILE}.new" "$LISTFILE"
 fi
+sed -i -e "s|/usr/lib/debug|/usr/share/debug|g" $LISTFILE
+sed -i -e "s|/usr/src/debug|/usr/share/debug/src|g" $LISTFILE
